### ISSUE TITLE
Revisjon av innhold - Skjemakatalog

### ISSUE THREAD

**Issue by anettepettersen** [Altinn/portal#1]
Mye av innholdet på skjemakatalog er utdatert eller mangelfull. Dette må rettes opp i. Vedlagt ligger rapport utført av ABS som viser hvilket innhold som må rettes hvor, samt et innsiktsarbeid levert fra design. 

[Prosjekt_Skjemakatalog_Sluttrapport_Vår_2021.docx](https://github.com/Altinn/portal/files/8160962/Prosjekt_Skjemakatalog_Sluttrapport_Var_2021.docx)

[Kopi_av_Om_skjemasidene_dok.xlsx](https://github.com/Altinn/portal/files/8160932/Kopi_av_Om_skjemasidene_dok.xlsx)

**Comment by anettepettersen** [comment]
@ferkinx Setter du opp en plan for videre arbeid her?

**Comment by JFerkingstad** [comment]
Ja, holder på å planlegge. De som har vedlikeholdt skjemakatalogen de siste årene, er vel Narvik og servicedesken?

**Comment by anettepettersen** [comment]
@ferkinx Ja, til den grad det har vært vedlikeholdt, så er det vel det. Legger du inn planen her etter hvert? Du kan redigere og legge til opprinnelig saksbeskrivelse. Si i fra om du trenger hjelp.
